python/ad-hoc-ingest/dataset_validator.py

```diff
@@ -1,14 +1,18 @@
 import os
+import sys
+import time
 from pathlib import Path
 
 import pandas
+import message_printer
```

Comment:
Maybe to shorten it in the code you could either: 

1. add `as` to shorten it e.g `message_printer as printer` or `message_printer as mp`
2. or change this to  `from message_printer import progress, message` and then use the functions instead?